seed_isort_config.py

```diff
@@ -122,21 +122,20 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
         else:
             load = ini_load
             dump = ini_dump
-
-        with open(filename, encoding='UTF-8') as f:
```

Comment:
could you leave the blank line above the `with` as it was in the left hand side -- thanks!

Reply:
fixed